proof/refine/Move_R.thy

```diff
@@ -250,4 +250,105 @@ lemma check_active_irq_invs_just_idle:
         and (\<lambda>s. 0 < domain_time s) and valid_domain_list \<rbrace>"
   by (wpsimp simp: check_active_irq_def ct_in_state_def)
 
+(* FIXME RT: Move? is this usable? *)
```

Comment:
I reckon keep it for now. I still have a suspicion there might be a generalisation, but haven't had any time to think about it.